test/services/CompanyDetailsServiceSpec.scala

```diff
@@ -206,4 +206,48 @@ class CompanyDetailsServiceSpec extends AnyWordSpecLike with ErsTestHelper {
       result shouldBe ()
     }
   }
+
+  "deleteCompany" must {
+    val companyDetailsService: CompanyDetailsService = new CompanyDetailsService(mockErsUtil, mockSessionService)
+
+    "return true" when {
+      "delete operation was successful with multiple companies" in {
```

Comment:
can also maybe cut down on code reuse in test files? (see `test/controllers/GroupSchemeControllerSpec.scala` comments)

Reply:
Most of these tests are just refactored ones of the originals to match the rest of the connectorSpecs layouts. I have joined up 2 of the tests, if more need joining it's probably best saved for the testing tickets coming out of the test Spikes Accenture are handling currently I'd say